cosmos/lib/cosmos/script/calendar.rb

```diff
@@ -28,9 +28,9 @@ module Script
     # Sets the metadata value for a target
     #
     # @param target [String] Target to set metadata on
-    # @return The result of the method call. 
+    # @return The result of the method call.
     def get_metadata(target)
-      endpoint = "/cosmos-api/metadata/get/#{target}"
+      endpoint = "/cosmos-api/metadata/_get/#{target}"
```

Comment:
Why _get?

Reply:
I changed the route a while back, I forgot to update the script work though.

Reply:
Ok, but why the underscore.  That would typically indicate some sort of private endpoint and this isn't private.

Reply:
Hmmm I didn't know that, I put the underscore as a way to distinguish the endpoint from the get endpoint for an individual metadata object.